modules/tinymce/src/core/main/ts/api/commands/ClipboardCommands.ts

```diff
@@ -1,38 +1,102 @@
+import { PlatformDetection } from '@ephox/sand';
+
 import Editor from '../Editor';
 import Env from '../Env';
 
 export const registerCommands = (editor: Editor): void => {
-  editor.editorCommands.addCommands({
-    'Cut,Copy,Paste': (command) => {
-      const doc = editor.getDoc();
-      let failed;
+  const browser = PlatformDetection.detect().browser;
+
+  const copyToClipboard = async (text: string) => {
+    await navigator.clipboard.writeText(text);
+  };
+
+  const pasteFromClipboard = async (): Promise<void> => {
+    let text = '';
+    text = await navigator.clipboard.readText();
+    editor.selection.setContent(text);
+  };
+
+  const cutToClipboard = async (text: string) => {
+    await navigator.clipboard.writeText(text);
+    editor.selection.setContent('');
+  };
+
+  const handleSecureContentError = () => {
+
+    let msg = editor.translate(
+      'Error: This operation requires a secure context (HTTPS)' +
+        'Direct access to the clipboard is not allowed in insecure contexts.'
```

Comment:
It's not on two lines in the rendering, so punctuation is neccessary
```suggestion
      'Error: This operation requires a secure context (HTTPS).' +
        'Direct access to the clipboard is not allowed in insecure contexts.'
```